enterprise/resign/resignOwncloudApp

```diff
@@ -58,11 +58,12 @@ APPPAYLOAD="$APPTEMP/Payload"
 APPDIR="$APPPAYLOAD/ownCloud.app"
 PROVISIONING_DIR="Provisioning Files"
 
-DISTRIBUTION_MOBILEPROVISION="$PROVISIONING_DIR/App.mobileprovision"
-FILEPROVIDER_MOBILEPROVISION="$PROVISIONING_DIR/FileProvider.mobileprovision"
-FILEPROVIDERUI_MOBILEPROVISION="$PROVISIONING_DIR/FileProviderUI.mobileprovision"
-INTENT_MOBILEPROVISION="$PROVISIONING_DIR/Intent.mobileprovision"
-SHAREEXTENSION_MOBILEPROVISION="$PROVISIONING_DIR/ShareExtension.mobileprovision"
+DISTRIBUTION_MOBILEPROVISION="$PROVISIONING_DIR/$METHOD/App.mobileprovision"
+FILEPROVIDER_MOBILEPROVISION="$PROVISIONING_DIR/$METHOD/FileProvider.mobileprovision"
+FILEPROVIDERUI_MOBILEPROVISION="$PROVISIONING_DIR/$METHOD/FileProviderUI.mobileprovision"
+INTENT_MOBILEPROVISION="$PROVISIONING_DIR/$METHOD/Intent.mobileprovision"
+SHAREEXTENSION_MOBILEPROVISION="$PROVISIONING_DIR/$METHOD/ShareExtension.mobileprovision"
+ASSOCIATED_DOMAINS=$(<$PROVISIONING_DIR/$METHOD/domains.txt)
```

Comment:
`$METHOD/` is no longer available in the current version. Please remove this part from the path.